docs/Deploying-Pega-on-AKS.md

```diff
@@ -409,6 +410,55 @@ Use the provided example addons.yaml file to configure the use of an Application
 
 3. Save the file.
 
+
+### Updating the backingservices.yaml Helm chart values (Applicable only when installing or upgrading to Pega 8.6 and later)
+
+
+Adding the Pega configuration files to your Helm installation on your local system
+----------------------------------------------------------------------------------
+Pega maintains a repository of Helm charts that are required to deploy Pega Platform and related resources using Helm, including a generic version of the following charts. After you add the repository to your local system, you can customize these configuration files for your backing service deployment:
+
+- pega/backingservices - Use this chart to set customization parameters for your deployment. You will modify this chart later in the deployment tasks.
+
+1. To add the backingservices repository to your Helm installation, enter:
+
+    `$ helm repo add pega https://dl.bintray.com/pegasystems/pega-helm-charts`
+    
+2. To verify the new repository, you can search it by entering:
+    
+```bash
+$ helm search repo pega
+NAME                	CHART VERSION	APP VERSION	DESCRIPTION                    
+pega/pega           	1.4.4        	           	Pega installation on kubernetes
+pega/addons         	1.4.4        	1.0        	A Helm chart for Kubernetes    
+pega/backingservices	1.4.4        	           	A Helm chart for Kubernetes
```

Comment:
Helm Chart to provision the latest Search and Reporting Service (SRS) for your Pega Infinity deployment.